added AL, FL, and PL for seed status (#19)
praetorian-inc/praetorian-cli: README.md

The CLI has a plugin engine for you to extend the CLI without changing its internals. Your script
is imported to the CLI context so it has full and authenticated access to the SDK.

To run a script, add the `--script` option after the CLI command, for example:

```zsh
$ praetorian chariot list seeds --script ~/code/my-process-seeds.py
```

> The CLI [ships with scripts](https://github.com/praetorian-inc/praetorian-cli/tree/main/praetorian_cli/scripts) for
> common tasks. For those, you only need the script name to invoke them:
> 
> ```zsh
> $ praetorian chariot get seed 'SEED_KEY' --script list-assets
> ```

Reviewer: ```suggestion
For built in [scripts](https://github.com/praetorian-inc/praetorian-cli/tree/main/praetorian_cli/scripts) you only need the script name:
```zsh
$ praetorian chariot get seed 'SEED_KEY' --script list-assets
```
```

> To work with the plugin engine, the script needs to implement a `process` function that takes 4 arguments. They are:

Reviewer: ```suggestion
To work with the plugin engine, the script needs to implement a `process` function that takes 4 arguments:
```
